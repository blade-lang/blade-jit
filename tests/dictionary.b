var dict = {
  name: 'Richard',
  address: 'Plot 10, Alagbaa Estate',
  age: 28,
  married: true,
  children: 1,
}

echo dict

echo dict['address']
echo dict['age'] = 30

dict['children'] += 1

echo dict

var name = 'Richard'
var age = 30

echo {name, age,}
echo {name, age: 53,}
echo {name: 'Alexander', age,}
