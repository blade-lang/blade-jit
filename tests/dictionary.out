{name: Richard, address: Plot 10, Alagbaa Estate, age: 28, married: true, children: 1}
Plot 10, Alagbaa Estate
30
{name: Richard, address: Plot 10, Alagbaa Estate, age: 30, married: true, children: 2}
{name: Richard, age: 30}
{name: Richard, age: 53}
{name: Alexander, age: 30}